catalog/core/catalog-core-migratable/src/main/java/org/codice/ddf/catalog/migratable/impl/CatalogMigratableImpl.java

```diff
@@ -50,22 +49,18 @@
  * Implementation of the {@link org.codice.ddf.migration.Migratable} interface used to migrate
```

Comment:
`CatalogMigratableImpl` needs to do both, extend from `AbstractDescribable` **and implement `DataMigratable`**. We should also update the comment as @figliold mentioned.